components/dashboard/src/components/DropDown2.tsx

```diff
@@ -0,0 +1,201 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import React, { FunctionComponent, RefObject, useCallback, useEffect, useMemo, useRef, useState } from "react";
+import Arrow from "./Arrow";
+
+export interface DropDown2Element {
+    id: string;
+    element: JSX.Element;
+    isSelectable?: boolean;
+}
+
+export interface DropDown2Props {
+    getElements: (searchString: string) => DropDown2Element[];
+    searchPlaceholder?: string;
+    disableSearch?: boolean;
+    expanded?: boolean;
+    onSelectionChange: (id: string) => void;
+}
+
+export const DropDown2: FunctionComponent<DropDown2Props> = (props) => {
```

Comment:
Non-blocking - but I think it would be helpful in the future to have a better name for this component to help explain what it's intended for, or know when to use this vs. `Dropdown`. I don't have any good suggestions though, but thought I'd mention it in case you had any ideas.